wmn-data.json

```diff
@@ -4280,6 +4280,16 @@
         "known" : ["ftdl", "ducensor"],
         "cat" : "social"
        },
+       {
+        "name" : "Poe.com",
+        "uri_check" : "https://poe.com/profile/{account}",
+        "e_code" : 200,
+        "e_string" : "\"__isNode\": \"PoeUser\"",

```

Comment:
Can you please remove the space between the `: \"` so this is `\"__isNode\":\"PoeUser\"`

Reply:
In both commandline curl and web browser, I see no space.

Reply:
Removed the spaces, and in m_string as well